recipe/meta.yaml

```
@@ -1,46 +1,61 @@
 {% set name = "google-api-python-client" %}
-{% set version = "2.4.0" %}
+{% set version = "2.154.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 72f69e67238e38b0da41b5a2e352dc39b4b5891d88f3db082aae7775da8fe58b
+  url: https://github.com/googleapis/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d22e6bd0da4d851a2b04655743fb380d4b846db811b12ef831faa93caf23b87d
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
```

```suggestion
  skip: True  # [py<37]
```
https://github.com/googleapis/google-api-python-client/blob/v2.154.0/setup.py#L72
We do not build below 3.9, but our recipes are open-source. Might as well make it follow upstream.